x_coord: 5.0
y_coord: 6.0
z_coord: 5.0
yaw_des: 0.0
radius: 0.5
min_dist: 0.3
arm_length: 0.53
rate: 10.0
exp_r: 2.0
repel_const: 1.0